clustermesh/clustermesh.go

```diff
@@ -466,6 +469,7 @@ type Parameters struct {
 	ConfigOverwrites     []string
 	Retries              int
 	HelmValuesSecretName string
+	Output               string
```

Comment:
Do you mean an enum type like this?

```go
type ParameterOutput int

const (
   Json ParameterOutput = iota
  Summary
)
```

Reply:
Yeah, something like that. I didn't consider the implications wrt. `StringVarP`. If this turns out too cumbersome to change, then let's leave it as is.

Reply:
Thinking about it again, could we at least define the constants as plain strings and use them rather than repeating `"json"` and `"summary"` several times, i.e.

```go
const (
  OutputJSON = "json"
  OutputSummary = "summary"
)